lib/cogserver.conf

```diff
@@ -73,29 +73,29 @@ ANSI_ENABLED	       = true
 
 # Cogserver in OSX will automatically change .so extension to .dylib
 # if .so exists.
-MODULES               = opencog/modules/libbuiltinreqs.so,
-                        opencog/modules/libnlp-types.so,
-                        opencog/modules/libattention-types.so,
-                        opencog/modules/libPLNTypes.so,
-                        opencog/modules/libembodiment-types.so,
-                        opencog/modules/libPersistModule.so,
-                        opencog/modules/libPersistZmqModule.so,
-                        opencog/modules/libQueryModule.so,
-                        opencog/modules/libLGDictModule.so,
-                        opencog/modules/libSuRealModule.so,
-                        opencog/modules/libattention.so,
-                        opencog/modules/libAtomSpaceExtensions.so,
-                        opencog/modules/libPythonModule.so,
-                        opencog/modules/libpy-shell.so,
-                        opencog/modules/libscheme-shell.so,
-                        opencog/modules/libRuleEngineModule.so
+MODULES               = opencog/cogserver/modules/libbuiltinreqs.so,
+                        opencog/cogserver/modules/libnlp-types.so,
+                        opencog/cogserver/modules/libattention-types.so,
+                        opencog/cogserver/modules/libPLNTypes.so,
+                        opencog/cogserver/modules/libembodiment-types.so,
+                        opencog/cogserver/modules/libPersistModule.so,
+                        opencog/cogserver/modules/libPersistZmqModule.so,
+                        opencog/cogserver/modules/libQueryModule.so,
+                        opencog/cogserver/modules/libLGDictModule.so,
+                        opencog/cogserver/modules/libSuRealModule.so,
+                        opencog/cogserver/modules/libattention.so,
+                        opencog/cogserver/modules/libAtomSpaceExtensions.so,
+                        opencog/cogserver/modules/libPythonModule.so,
+                        opencog/cogserver/modules/libpy-shell.so,
+                        opencog/cogserver/modules/libscheme-shell.so,
+                        opencog/cogserver/modules/libRuleEngineModule.so
 
```

Comment:
argh. Right. what a mess. When I run "production" servers, e.g. for the behavior trees, the build directory is nowhere nearby, so everything has to run from the install dir.   For unit testing, it should test what is in the build dir, else things get confusing.